src/front-end/typescript/lib/pages/opportunity/code-with-us/edit/tab/summary.tsx

```diff
@@ -236,5 +245,33 @@ export const component: Tab.Component<State, InnerMsg> = {
   view,
   onInitResponse(response) {
     return adt("onInitResponse", response);
+  },
+  // Add "View Complete Competition" button action if the opportunity is awarded and the user is an admin
+  getActions: ({ state }) => {
+    const opportunity = state.opportunity;
+    const viewerUser = state.viewerUser;
+    if (
+      !opportunity ||
+      !isAdmin(viewerUser) ||
+      opportunity.status !== CWUOpportunityStatus.Awarded
+    ) {
+      return component_.page.actions.none();
+    }
+    return component_.page.actions.links([
+      {
+        children: "View Complete Competition",
+        symbol_: leftPlacement(iconLinkSymbol("external-link")),
+        button: true,
+        color: "primary" as const,
+        onClick: () => {
```

Comment:
You should be able to use the dest prop with the newTab option instead of this: https://github.com/bcgov/digital_marketplace/blob/development/src/front-end/typescript/lib/views/link.tsx#L114